src/components/Accordion/index.tsx

```diff
@@ -26,40 +27,40 @@ export const Accordion: React.VFC<AccordionProps> = ({
   titleTag,
   onToggle = () => {},
   useIndent = true,
+  isOpen,
```

Comment:
These changes weren't necessary `defaultOpen` would have worked.

Reply:
if we set defaultOpen, the chart will rerender every time new chart data is loaded. You can see this from Minna's video: https://github.com/evergreen-ci/spruce/pull/1212#discussion_r848544519